main.py

```diff
@@ -1,10 +1,43 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_data = json.load(f)
+    for key, value in players_data.items():
+        race = Race.objects.get_or_create(
+            name=value["race"]["name"],
+            description=value["race"]["description"]
+        )
+        guild = (None, None)
+        if value["guild"]:
+            guild = Guild.objects.get_or_create(
+                name=value["guild"]["name"],
+                description=value["guild"]["description"],
+            )
```

Comment:
Perform unpacking here and on line 11 so that there would be no need to retrieve value by index on lines 38 and 39
```suggestion
        if value["guild"]:
            guild, _ = Guild.objects.get_or_create(
                name=value["guild"]["name"],
                description=value["guild"]["description"],
            )
```